I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/GenderQuestion.rpy", line 106, in script
    e "[They] is a [attractive] [sex]."
  File "game/Character.rpy", line 11, in __str__
    if pronouns == "she/her":
NameError: name 'pronouns' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/GenderQuestion.rpy", line 106, in script
    e "[They] is a [attractive] [sex]."
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\ast.py", line 621, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\exports\sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\character.py", line 1440, in __call__
    what = self.prefix_suffix("what", self.what_prefix, what, self.what_suffix)
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\character.py", line 1354, in prefix_suffix
    return (sub(prefix) + sub(body) + sub(suffix))
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\character.py", line 1334, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\substitutions.py", line 373, in substitute
    s = interpolate(s, variables) # type: ignore
  File "C:\Program Files (x86)\renpy-8.3.4-sdk\renpy\substitutions.py", line 97, in interpolate
    rv += format(value, fmt)
  File "game/Character.rpy", line 11, in __str__
    if pronouns == "she/her":
NameError: name 'pronouns' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.4.24120703
TBN 1.0
Sun Feb 23 16:22:07 2025
